frontend/src/v5/store/users/users.selectors.ts

```diff
@@ -30,9 +31,14 @@ export const selectUsersByTeamspace = createSelector(
 
 export const selectUser = createSelector(
 	selectUsersByTeamspace,
-	(_, teamspace, userName) => userName,
-	(usersInTeamspace, userName): IUser | null => usersInTeamspace
-		.find((teamspaceUser) => teamspaceUser.user === userName),
+	(_, teamspace, userName) => [teamspace, userName],
+	(usersInTeamspace, [teamspace, userName]): IUser => {
+		const user = usersInTeamspace.find((teamspaceUser) => teamspaceUser.user === userName);
+		if (user) {
+			return { ...user, avatarUrl: getMemberImgSrc(teamspace, user.user), hasAvatar: true };
+		}
+		return { user: userName, firstName: userName, lastName: '', avatarUrl: '' };
```

Comment:
Can we add a field stating that the user is not a member of the teamspace?
and then change the condition in here to ``src\v5\ui\components\shared\userPopover\userPopover.component.tsx``

<img width="608" alt="image" src="https://user-images.githubusercontent.com/42032433/217614620-95984e18-1963-4550-82c0-4677a201117f.png">

Reply:
Done